move/interchain_token_service/sources/interchain_token_service.move

```diff
@@ -474,6 +478,12 @@ module interchain_token_service::interchain_token_service {
         );
     }
 
+    public(package) fun receive_link_coin<T>(self: &mut InterchainTokenService, approved_message: ApprovedMessage) {
+        let value = self.value_mut!(b"receive_link_coin");
+
+        value.receive_link_coin<T>(approved_message);
+    }
```

Comment:
I moved this to be with the other package functions (it was with the public functions). However, I suspect this is supposed to be a `public fun`, not `public(package)` 
@milapsheth can you confirm?